junction/static/less/app.less

```diff
@@ -0,0 +1,10 @@
+@import "vendor.less";
+
+@import "variables.less";
+@import "bootstrap_overrides.less";
+
+@import "utils.less";
+@import "base.less";
+
+@import "editor.less";
+@import "home.less";
```

Comment:
Nice job with integrating a pre-processor. :+1: 

Not saying `.less` is bad or anything. wondering if `scss` can be an option here. It seems to have better extensibility with mixins and inheritance. It is being used for python.org (https://github.com/python/pythondotorg/tree/master/static). I'm happy with with both as they solve quite a lot of core issues that are present with css. thoughts?